src/scenes/Dashboard/scenes/ChangeState/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import {
+  Col,
+  Row,
+  Typography,
+  Card,
+  Steps,
+  Button,
+  Progress,
+  Modal,
+} from 'antd';
+import { State } from './interfaces';
+import { ExclamationCircleOutlined } from '@ant-design/icons';
+import styles from './styles.css';
+
+const { Title, Text } = Typography;
+const { Step } = Steps;
+const programStates: State[] = [
+  {
+    stepNo: 0,
+    name: 'Created',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet urna id diam suscipit ultricies eget in lorem.',
+  },
+  {
+    stepNo: 1,
+    name: 'Mentor Applicaiton Period',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet urna id diam suscipit ultricies eget in lorem. ',
+  },
+  {
+    stepNo: 2,
+    name: 'Mentor Approval Period',
+    description:
+      'Some descriptive text about this particular state of the program',
+  },
+  {
+    stepNo: 3,
+    name: 'Mentee Applicaiton Period',
+    description:
+      'Some descriptive text about this particular state of the program. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet urna id diam suscipit ultricies eget in lorem. Nulla at lorem a eros mattis iaculis et in dolor.',
+  },
+  {
+    stepNo: 4,
+    name: 'Mentee Approval Period',
+    description:
+      'Some descriptive text about this particular state of the program',
+  },
+  {
+    stepNo: 5,
+    name: 'Ongoing',
+    description:
+      'Some descriptive text about this particular state of the program',
+  },
+];
+
+const ChangeState = () => {
```

Comment:
```suggestion
function ChangeState(){
```

Just to improve the readability

Reply:
I'm on it!

Reply:
Resolved with commit https://github.com/sef-global/scholarx-frontend/pull/13/commits/b3f5828504720c39c8fe49b514b6e216c682eacb. Please review it again.